example.env.tt

```diff
@@ -27,3 +27,9 @@ PORT=3000
 # SENTRY_DSN=http://public@example.com/project-id
 SENTRY_CSP_HEADER_REPORT_ENDPOINT=https://SOMECODE.ingest.sentry.io/api/SOMENUMS/security/?sentry_key=SOMETHING
 SENTRY_ENV=development
+
+# Add a meta tag to your HTML in staging env which will stop google indexing it.
+# Default is YES, when production env, it should be NO
+#
+# https://developers.google.com/search/reference/robots_meta_tag
+NO_INDEX=YES
```

Comment:
I wonder if a longer name like `PREVENT_SEARCH_ENGINE_INDEXING` would be more explanatory?

Reply:
Thank you